src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -710,9 +738,27 @@ public void townToggle(Player player, String[] split) throws TownyException {
 			} else if (split[0].equalsIgnoreCase("pvp")) {
 				// Make sure we are allowed to set these permissions.
 				toggleTest(player, town, StringMgmt.join(split, " "));
-				town.setPVP(!town.isPVP());
-				TownyMessaging.sendTownMessage(town, String.format(TownySettings.getLangString("msg_changed_pvp"), town.getName(), town.isPVP() ? "Enabled" : "Disabled"));
-
+				boolean outsiderintown = false;
+				if (!TownySettings.getOutsidersPreventPVPToggle()) {

```

Comment:
if (TownySettings.getOutsidersPreventPVPToggle()) { 

- remove !

Reply:
Since we use prevent then we want the config option to be false, right if we want to keep the default values and this as an added feature people can enable?

Reply:
By changing the wording to prevent we would need to remove the ! but then also change the config option to false

Reply:
Which we can do there's no problem just that the wording is a little bit messy if we change it